pkg/datapath/linux/node.go

```diff
@@ -459,8 +459,17 @@ func upsertIPsecLog(err error, spec string, loc, rem *net.IPNet, spi uint8) {
 	}
 }
 
+func getDefaultEncryptionInterface() string {
+	iface := ""
+	if len(option.Config.EncryptInterface) > 0 {
+		iface = option.Config.EncryptInterface[0]
```

Comment:
This works for EKS because it doesn't seem to care what interface we use. I was going to fix this up in another series to do this properly after I decide what that means. I'm not sure why we are hard coding this right now.

Reply:
I added a comment to the code, but it feels a bit lame. Anyone want to object loudly to pushing the further cleanup/fix to a follow up PR. I'm a bit hesitant to start doing major refactors here until I have CI in place to test them on the various envs (EKS, AKS, GKE, etc.) and can't get those down until this PR is merged.